index.bs

```diff
@@ -2453,6 +2453,8 @@ Unanimous Short Circuit</h5>
 	the Team recommends a resolution
 	and every potential member of a Council who is not renouncing their seat
 	votes affirmatively (no abstentions) to adopt this resolution.
+	As an exception, Tim Berners-Lee <em class=rfc2119>may</em> abstain
+	without blocking the adoption of the resolution.
```

Comment:
```suggestion
	and every potential member of a Council who is not renouncing their seat
	(with the possible exception of Tim Berners-Lee)
	votes affirmatively (no abstentions,
	again with the possible exception of Tim Berners-Lee)
	to adopt this resolution.
```

Reply:
This edit no longer makes sense, since the underlying text has been changed by #885.